docs/deployment/resource-providers/mesos.md

```diff
@@ -26,55 +26,48 @@ under the License.
 * This will be replaced by the TOC
 {:toc}
 
-## Background
+## Getting Started
```

Comment:
In my mind, the `Getting Started` section should be an opinionated (make choices for the user) and short (ideally doable in 15 minutes) path to having a running Flink cluster.

In particular, I would shorten the Introduction by removing the detailed description of the components, and move them to the reference.
Secondly, I would only describe how to set up on dcos, and again move the manual setup to the reference. This way, it'll just be a few commands and little text to read to success.